lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -7,55 +7,83 @@ local std = require "nvim-lsp-installer.core.managers.std"
 
 local coalesce, when = functional.coalesce, functional.when
 
+function GenerateCmd(mono_path, root_dir)
+  if mono_path == nil and vim.fn.executable("mono") then
+    mono_path = "mono"
+  end
```

Comment:
I don't see a reason to take a `mono_path` as a setting? Wouldn't it be enough to assume it is on path, and instead let the user decide via a boolean flag, like

```lua
lspconfig.omnisharp.setup {
  use_mono = true
}
```

Or am I missing something here?